packages/service-clients/odsp-client/src/interfaces.ts

```diff
@@ -79,6 +82,23 @@ export interface OdspContainerAttachProps {
 	fileName: string | undefined;
 }
 
+/**
+ * Events emitted by the ODSP container service to notify consumers of select
+ * container changes.
+ * @beta
+ * @sealed
+ */
+export interface IOdspContainerServicesEvents extends IEvent {
+	/**
+	 * Emitted when the read-only state of the container changes.
+	 */
+	(event: "readOnlyStateChanged", listener: (readonly: boolean) => void): void;
```

Comment:
For both of these, if they can be directly retrieved imperatively from their corresponding methods then prefer to eliminate the params here.

Reply:
Ah, that makes sense. I can add doc to call those methods to retrieve the latest information if wanted